packages/storefront/src/pages/patterns/forms/resources.md

```diff
@@ -92,7 +84,7 @@ Below you can find the explanations of the used class names.
         </p-fieldset>
         <p-button-group class="form-section-spacing form-bottom-spacing">
           <p-button type="submit">Send</p-button>
-          <p-button variant="tertiary" icon="close"type="reset">Cancel</p-button>
+          <p-button variant="tertiary" icon="none" type="reset">Cancel</p-button>
```

Comment:
Change `tertiary`to `secondary` and delete `icon="none"` because no icon is the default.